<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KifuAnalysisDialog</class>
 <widget class="QDialog" name="KifuAnalysisDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>690</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>棋譜解析</string>
  </property>
  <widget class="QGroupBox" name="groupBox_6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>621</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>範囲</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>601</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_12">
       <item>
        <widget class="QRadioButton" name="radioButtonInitPosition">
         <property name="text">
          <string>開始局面から</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_13">
       <item>
        <widget class="QRadioButton" name="radioButtonCurrentPosition">
         <property name="text">
          <string>現在局面から</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>621</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>エンジン設定</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>601</width>
      <height>95</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QComboBox" name="comboBoxEngine1"/>
       </item>
       <item>
        <widget class="QPushButton" name="engineSetting">
         <property name="text">
          <string>エンジン設定</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>340</y>
     <width>621</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>時間制限</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>601</width>
      <height>95</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>１手あたりの思考時間</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="byoyomiSec"/>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>秒</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>510</y>
     <width>228</width>
     <height>35</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
